### ISSUE TITLE
Publish Current Version Info Only in tools/currentversion.sh & tools/currentversion.ps1

### ISSUE THREAD

**Issue by DarwinJS** [PowerShell/PowerShell#3519]
I am writing some scripts that need to pull the latest version of PowerShell (whether or not they are setup in a repo) - similar to tools/download.sh.

It would be great if you published something like tools/currentversion.sh (and currentversion.ps1) that contained:
release=v6.0.0-alpha.18
build=18

Then I could download and run that shell script in my code (at my own risk) and be able to pull the current version.

Other data might be helpful as well (maybe actual package files per platform target and their checksums?) - but at a minimum being able to auto-detect the current version would be very helpful!

**Comment by iSazonov** [comment]
Related Issues #1635 #1395 
@DarwinJS Please clarify what do you use "current version" for? I believe that preferred way is always to test the availability of features.

**Comment by DarwinJS** [comment]
@iSazonov - it would allow me to build something similar to download.sh - which pulls the latest version of PowerShell for the target environment - but I would be in control of the details of what happens in the download.sh equivalent.

So within my script I would call something similar to this:
```
wget -O latestsversion.sh https://raw.githubusercontent.com/PowerShell/PowerShell/master/tools/latestversion.sh
. latestversion.sh
```
For languages other than bash, the file could be pulled and parsed as key value pairs.

**Comment by iSazonov** [comment]
Can you use [nuget](http://stackoverflow.com/questions/11867660/list-all-available-versions-of-a-specific-package-in-nuget-package-manager-conso)?

**Comment by DarwinJS** [comment]
@iSazonov - I wouldn't be interested in using nuget for linux.  As far as Windows, I am already the maintainer for the Chocolatey package for PowerShell core: https://chocolatey.org/packages/powershell-core

**Comment by joeyaiello** [comment]
I'm not sure that it makes sense to have yet another script to manually update every time we do a release. 

`(Invoke-RestMethod https://api.github.com/repos/powershell/powershell/releases/latest).tag_name` should give you the latest version string today. If you need something else that's not provided there, I'd love to get some more info.

**Comment by DarwinJS** [comment]
@joeyaiello - this script is to bootstrap PowerShell itself so I can't use invoke-restmethod - do you know a simple a bash equivalent using curl or wget?  I think wget may be more prevalent - I am having to install it on Ubuntu.

**Comment by joeyaiello** [comment]
`curl` will get you the JSON block just fine, but then you have to parse it. It's relatively painless if you've got Python, but if you're stuck with basic coreutils, check this out: http://stackoverflow.com/questions/1955505/parsing-json-with-unix-tools

**Comment by DarwinJS** [comment]
Great link - I was able to craft a solution based on a pure curl + sed example!

**Comment by iSazonov** [comment]
@DarwinJS It seems #3521 is related.

**Comment by DarwinJS** [comment]
Did a PR to make download.sh detect version rather than have hard codes: https://github.com/PowerShell/PowerShell/pull/3535

**Comment by iSazonov** [comment]
The Issue was resolved in #3608

**Comment by DarwinJS** [comment]
The best resolution is actually in: https://github.com/PowerShell/PowerShell/issues/3586
